lib/msal-node/test/client/ClientCredentialClient.spec.ts

```diff
@@ -401,6 +401,85 @@ describe("ClientCredentialClient unit tests", () => {
         checkMockedNetworkRequest(returnVal, checks);
     });
 
+    it("Validates that claims and client capabilities are correctly merged", async () => {
+        sinon
+            .stub(Authority.prototype, <any>"getEndpointMetadataFromNetwork")
+            .resolves(DEFAULT_OPENID_CONFIG_RESPONSE.body);
+        sinon
+            .stub(
```

Comment:
In the future, can you use HTTP mocking in tests? Do you feel stubs / sinon is easier than mocking just the HTTP responses?

Reply:
No, mocking is easier. I used sinon because that's what all tests currently use. I'll switch to mocking http responses from now on, unless I need to use sinon's "spy".